mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -462,11 +444,70 @@ def test_read_vmrk_annotations():
         for item in head:
             temp.write(item)
         temp.seek(0)
-        annotations = read_annotations(temp.name, sfreq=sfreq)
+        read_annotations(temp.name, sfreq=sfreq)
     try:
         temp.close()
         unlink(temp.name)
     except FileNotFoundError:
         pass
 
+
+@testing.requires_testing_data
+def test_read_vhdr_annotations_and_events():
+    """Test load brainvision annotations and parse them to events."""
+    sfreq = 1000.0
+    expected_orig_time = 1384359243.794231
+    expected_onset_latency = np.array(
+        [0, 486., 496., 1769., 1779., 3252., 3262., 4935., 4945., 5999., 6619.,
+         6629., 7629., 7699.]
+    )
+    expected_annot_description = [
+        'New Segment/', 'Stimulus/S253', 'Stimulus/S255', 'Stimulus/S254',
+        'Stimulus/S255', 'Stimulus/S254', 'Stimulus/S255', 'Stimulus/S253',
+        'Stimulus/S255', 'Response/R255', 'Stimulus/S254', 'Stimulus/S255',
+        'SyncStatus/Sync On', 'Optic/O  1'
```

Comment:
Might also include an `S 10` in there to ensure it's parsed correctly.

Reply:
you mean like https://github.com/mne-tools/mne-python/pull/6326/commits/0f1759e1d606b992aef1f5bed5fb8a2f7bb92f38 ?